modVM/src/org/aion/vm/AbstractExecutionResult.java

```diff
@@ -38,19 +38,36 @@
     byte[] output;
 
     public enum ResultCode {
+        /*
+         * Indicates a successful transaction.
+         */
```

Comment:
minor formatting issue: missing the [plugin](https://github.com/aionnetwork/aion/wiki/Aion-Code-Conventions#setting-up-the-intellij-google-java-format-plugin)